vectorstores/milvus/milvus.go

```diff
@@ -18,7 +18,7 @@ import (
 type Store struct {
 	dropOld          bool
 	async            bool
-	initialized      bool
```

Comment:
The changes in this file seem unrelated to the main goal of the PR, or am I missing something? It'd be good to keep the PR for a single purpose only